### PR TITLE
Fix/guest agent

### PR DESCRIPTION
**Pull request by nyoxi**
Fixes links and formatting on couple of Guest Agent related pages.

### CODE REVIEW

File: source/documentation/internal/guest-agent/architecture.html.md [hunk 1]
```diff
@@ -14,16 +14,17 @@ wiki_last_updated: 2013-05-20
 
 ## The communication between oVirt guest agent and the oVirt engine
 
-![](Guest_Agent_Achitecture.png "Guest_Agent_Achitecture.png")
+<!-- TODO: image too large, needs fixing
```
Comment (mykaul) [inline source/documentation/internal/guest-agent/architecture.html.md]: We can probably fix if it's too big. Do you need help with that?

Comment (nyoxi) [inline source/documentation/internal/guest-agent/architecture.html.md]: No I can do that. It's just that I'm hessitant whether to keep the image at all. But I will shrink it and we can remove it later if we want.

**Comment by mykaul** [comment]
@JohnMarksRH - can you please review?

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 2]
```diff
@@ -19,38 +19,38 @@ It provides:
...
-      * `**`Power` `Up`**` - Send when agent start its execution.
-      * `**`Power` `Down`**` - unused
-      * `**`Heartbeat`**` - Message is send every few second to notify that the agent is running. The notification includes the guest's available RAM.
-      * `**`User` `Info`**` - Active user was changed.
-      * `**`Session` `Lock`**` - Desktop was locked (Windows).
-      * `**`Session` `Unlock`**` - Desktop was unlocked (Windows).
-      * `**`Session` `Logoff`**` - A user was logged off (Windows).
-      * `**`Session` `Logon`**` - A user was logged on (Windows).
-      * `**`Agent` `Uninstalled`**` - Agent was removed from system. Expected to be send from the agent's installer.
+* **Power up** -- Send when agent start its execution.
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: Corrected version (changes in **bold**)
* **Power up** -- Send when **the agent starts to execute.**

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 3]
```diff
@@ -19,38 +19,38 @@ It provides:
...
+* **Power down** -- unused
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: **Unused**

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 4]
```diff
@@ -19,38 +19,38 @@ It provides:
...
+* **Heartbeat** -- Message is send every few second to notify that the agent is running. The notification includes the guest's available RAM.
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: Corrected version (changes in **bold**)
* **Heartbeat** -- **The** message **sent** every few seconds to notify that the agent is running. The notification includes the guest's available RAM.

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 5]
```diff
@@ -19,38 +19,38 @@ It provides:
...
+* **User info** -- Active user was changed.
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: This is not clear

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 6]
```diff
@@ -19,38 +19,38 @@ It provides:
...
+* **Session lock** -- Desktop was locked (Windows).
+* **Session unlock** -- Desktop was unlocked (Windows).
+* **Session logoff** -- A user was logged off (Windows).
+* **Session logon** -- A user was logged on (Windows).
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: Probably should be "A user **is** logged on (Windows)."

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 7]
```diff
@@ -19,38 +19,38 @@ It provides:
...
+* **Agent uninstalled** -- Agent was removed from system. Expected to be send from the agent's installer.
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: Suggested edit (changes in **bold**)
* **Agent uninstalled** -- Agent was removed from **the** system. Expected to be **sent** from the agent's installer.

File: source/documentation/internal/guest-agent/guest-agent.html.md [hunk 8]
```diff
@@ -19,38 +19,38 @@ It provides:
...
 The following actions can be requested from the agent:
 
-      * `**`Lock` `screen`**` - Request locking the user's desktop.
-      * `**`Login`**` - Perform a login in user's behalf.
-` * `[`Automatic` `login` `on` `RHEL`](Ovirt_guest_agent_automatic_login_RHEL6)
-` * `[`Automatic` `login` `on` `Microsoft's` `Windows`](Ovirt_guest_agent_automatic_login_windows)
-      * `**`Logoff`**` - Log off the active user.
-      * `**`Shutdown`**` - Shut down the virtual machine.
+* **Lock screen** -- Request locking the user's desktop.
+* **Login** -- Perform a login in user's behalf.
```
Comment (JohnMarksRH) [inline source/documentation/internal/guest-agent/guest-agent.html.md]: Suggested edit (changes in **bold**)
* **Login** -- Perform a login **on the** user's behalf.

File: source/documentation/internal/guest-agent/architecture.html.md [hunk 9]
```diff
@@ -14,16 +14,15 @@ wiki_last_updated: 2013-05-20
...
+![](/images/wiki/Guest_Agent_Achitecture.png)
 
 The engine directs requests for the guest agent to VDSM.
 
 VDSM communicates with the guest using a VirtIO channel using plain text commands in JSON format. Each command is terminated by a new line character.
 
 ## VDSM/Guest Agent message structure
 
-Each message sent by VDSM has a mandatory field '__name__' which contains the command name.
+Each message sent by VDSM has a mandatory field `__name__` which contains the command name.
```
Comment (nyoxi) [inline source/documentation/internal/guest-agent/architecture.html.md]: Backticks are used in Markdown for inline code/keywords, etc. Comapre: 'this' and `this`

**Comment by nyoxi** [comment]
The intent of the pull request was to fix only the Markdown formating so that the pages are rendered correctly. I did not mean to touch the text. But since you already took the time to read it and point out the errors I will make the corrections.

**Comment by nyoxi** [comment]
ping

**Pull request merged**